packages/p2p-v2/src/pages/orders/screens/OrdersChat/OrdersChat.scss

```
@@ -0,0 +1,10 @@
+.p2p-v2-orders-chat {
+    border-radius: 4px;
+    border: 8px solid #f2f3f4;
+    width: 440px;
```

Does this need to be pixels @nada-deriv? 🤔 
```suggestion
    width: 44rem;
```